libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccount.h

```diff
@@ -168,6 +170,12 @@ NS_SWIFT_NAME(UserAccount)
  */
 - (nullable id)customDataObjectForKey:(id)key;
 
+/** Sets the user's photo.
+ @param photo The user photo, usually the thumbnail of the user.
+ @param completion Optional callback block invoked when the photo has been set
+ */
+- (void)setPhoto:(UIImage*_Nullable)photo completion:(void (^ __nullable)(void))completion;
```

Comment:
@jeanbovet Since we are adding a completion block, we should also care about the success/failure. What do you think?

Reply:
That's a great question! I did consider it but backtracked because I wasn't sure how you wanted to treat error. For example, in the current implementation, either error in deleting or storing the image will still set the user: should I change that as well to not assign the photo if storing the photo fails?

Reply:
@jeanbovet Since the error will be bubbled up to the app, it could choose to ignore or handle it if needed.

Reply:
Updated, let me know what you think